### DISCUSSION TITLE
Why such a high IN_MEMORY_TABLE value from duckdb_memory()?

### DISCUSSION THREAD

**Discussion by jessepanorama** [duckdb/duckdb#14802] [Q&A]
I'm trying to prioritize speed of access by using a local DuckDB table. I have tabular information stored in a parquet file on S3, and I'm pulling it down locally with the following:
```
CREATE TABLE my_local_table AS SELECT * FROM read_parquet('s3://#{bucket_name}/#{cloud_path}')
```
I'm noticing that RAM usage is unexpectedly high when running this code. When monitoring the change in memory usage for a docker container running this code, I'm seeing about a 350MB memory increase despite the parquet file weighing in at a much smaller 39MB. I used `FROM duckdb_memory()` immediately after this `CREATE TABLE` query to try to better understand this memory usage, and I see that `BASE_TABLE` has a value of 5.32MB, and `IN_MEMORY_TABLE` has a value of 343.93MB. All other values are 0. DuckDB docs explain that `BASE_TABLE` describes the memory usage of the buffer manager, but what about `IN_MEMORY_TABLE`? Is DuckDB's in-memory representation of the data in the parquet file simply ~10x higher since it's uncompressed or is there something else to know about this large difference?

**Comment by Maxxen** [comment]
DuckDB in-memory tables store data uncompressed, so I'd guess that is why the difference is so large. Parquet regularly reaches 10x compression ratios.
